networkx/algorithms/community/tests/test_community.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python
+from nose.tools import *
+import networkx as nx
+import collections
+
+
+def validate_communities(result, expected):
+    result = [tuple(x) for x in result]
+    assert_equal(collections.Counter(result), collections.Counter(expected))
```

Comment:
We wanted a way to test if two lists have exactly the same elements no matter the order.